repo/packages/M/marathon/0/config.json

```diff
@@ -0,0 +1,147 @@
+{
+  "type": "object",
+  "properties": {
+    "marathon/libprocess-ip": {
```

Comment:
Fair point. How about `mesos/bind-ip` then? Whatever we do, I think it's unnecessary to include `libprocess`.